lib/generators/boring/sidekiq/install/install_generator.rb

```diff
@@ -33,17 +42,36 @@ def add_sidekiq_routes
 
         say "Adding sidekiq routes", :green
 
+        if options[:authenticate_routes_with_devise]
+          route = <<~RUBY
+            authenticate :user do
+              mount Sidekiq::Web => '/sidekiq'
+            end
+
+          RUBY
+        else
+          route = "mount Sidekiq::Web => '/sidekiq'\n\n"
+        end
+
         inject_into_file "config/routes.rb",
                          "require 'sidekiq/web'\n\n",
                          before: "Rails.application.routes.draw do\n"
 
         inject_into_file "config/routes.rb",
-                         optimize_indentation(
-                           "mount Sidekiq::Web => '/sidekiq'\n\n",
-                           2
-                         ),
+                         optimize_indentation(route, 2),
                          after: "Rails.application.routes.draw do\n"
       end
+
+      def show_message
+        return if options[:skip_routes]
+
+        if options[:authenticate_routes_with_devise]
+          readme "README"
+        else
+          say "\nWe've added Sidekiq routes. Please protect it as necessary to suit your requirements.",
```

Comment:
Let's update to " ... ... Please protect it with authentication as necessary ... ... ..."